sbin/submitty_daemon_jobs/submitty_jobs/jobs.py

```diff
@@ -257,7 +257,7 @@ def run_job(self):
         logger.write_to_log(log_file_path, log_msg)
         #create paths
         try:
-            with open("/usr/local/submitty/config/submitty.json", encoding='utf-8') as data_file:
+            with open(os.path.join(INSTALL_DIR, "config", "submitty.json"), encoding='utf-8') as data_file:
                 CONFIG = json.loads(data_file.read())
```

Comment:
Remove this and just use the already available `INSTALL_DIR` or `DATA_DIR` variables for whatever path you want in this job.